native/src/utils/DatabaseConnector.ts

```diff
@@ -247,31 +249,26 @@ class DatabaseConnector {
 
   async _loadMetaCities(): Promise<MetaCitiesType> {
     const path = this.getMetaCitiesPath()
-    const fileExists: boolean = await BlobUtil.fs.exists(path)
+    const fileExists = await BlobUtil.fs.exists(path)
 
-    if (fileExists) {
-      try {
-        const citiesMetaJson: MetaCitiesJsonType = JSON.parse(await this.readFile(path))
-        return mapValues(citiesMetaJson, cityMeta => ({
-          languages: mapValues(
-            cityMeta.languages,
-            ({
-              last_update: jsonLastUpdate
-            }): {
-              lastUpdate: Moment
-            } => ({
-              lastUpdate: moment(jsonLastUpdate, moment.ISO_8601)
-            })
-          ),
-          lastUsage: moment(cityMeta.last_usage, moment.ISO_8601)
-        }))
-      } catch (e) {
-        log('An error occurred while loading cities from JSON', 'warning')
-        reportError(e)
-      }
+    if (!fileExists) {
+      return {}
     }
 
-    return {}
+    const citiesMetaJson = await this.readFile<MetaCitiesJsonType>(path)
+    return mapValues(citiesMetaJson, cityMeta => ({
```

Comment:
Why have you removed the `try ..catch`? I think it would be still nice to have this.

Reply:
No, otherwise operations that don't make sense if this fails will still be executed. This should be caught in some place higher up imo.

Reply:
A change I made though: If we detect that the data is corrupted, we delete the file now, otherwise we would never recover from it.